optimizely/helpers/condition.py

```diff
@@ -174,6 +174,10 @@ def is_finite(self, value):
       # numbers.Integral instead of int to accomodate long integer in python 2
       return False
 
+    if isinstance(value, bool):
```

Comment:
Doesn't this belong in https://github.com/optimizely/python-sdk/pull/146?

Reply:
Yeah, but this showed up only when I as able to run new audiences compat suite tests with this PR. Will move it into #146.